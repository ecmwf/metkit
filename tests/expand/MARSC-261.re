 retrieve,
    class=od,
    type=pf,
    stream=ef,
    expver=0001,
    repres=sh,
    levtype=sfc,
    param=167/165/166/164/228,
    date=20250416,
    time=0000,
    step=6/12,
    number=1/2,
    domain=g,
    resol=n128,
    area=70.5/-21.0/30.0/40.5,
    grid=1.5/1.5,
    target=data.reference