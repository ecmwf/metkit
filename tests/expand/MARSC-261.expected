 RETRIEVE,
    CLASS      = OD,
    TYPE       = PF,
    STREAM     = ENFO,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = SFC,
    PARAM      = 167/165/166/164/228,
    DATE       = 20250416,
    TIME       = 0000,
    STEP       = 6/12,
    NUMBER     = 1/2,
    DOMAIN     = G,
    TARGET     = data.reference,
    RESOL      = N128,
    AREA       = 70.5/-21/30/40.5,
    GRID       = 1.5/1.5